app-modules/service-management/src/Rules/ManagedServiceRequestType.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2016-2024, Canyon GBS LLC. All rights reserved.
+
+    Aiding App™ is licensed under the Elastic License 2.0. For more details,
+    see <https://github.com/canyongbs/aidingapp/blob/main/LICENSE.>
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Aiding App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    <https://www.canyongbs.com> or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+namespace AidingApp\ServiceManagement\Rules;
+
+use Closure;
+use Illuminate\Contracts\Validation\ValidationRule;
+use AidingApp\ServiceManagement\Models\ServiceRequestType;
+
+class ManagedServiceRequestType implements ValidationRule
+{
+    /**
+     * Run the validation rule.
+     *
+     * @param  \Closure(string): \Illuminate\Translation\PotentiallyTranslatedString  $fail
+     */
+    public function validate(string $attribute, mixed $value, Closure $fail): void
+    {
+        if (auth()->user()->hasRole('authorization.super_admin')) {
+            return;
+        }
+
+        $team = auth()->user()->teams()->first();
+
+        $isManager = ServiceRequestType::where('id', $value)
+            ->whereHas('managers', function ($query) use ($team) {
+                $query->where('teams.id', $team?->getKey());
+            })
+            ->orWhereHas('auditors', function ($query) use ($team) {
+                $query->where('teams.id', $team?->getKey());
+            })
```

Comment:
Let's remove the auditor check here. You should only be able to create ones you are a manager of.